api/src/main/java/keywhiz/api/LoginRequest.java

```diff
@@ -26,7 +26,7 @@
 @AutoValue
 public abstract class LoginRequest {
   @JsonCreator public static LoginRequest from(
-      @JsonProperty("username") String username, @JsonProperty("password") char[] password) {
+      @JsonProperty(value="username", required=true) String username, @JsonProperty(value = "password", required=true) char[] password) {
```

Comment:
I was chasing an NPE between jackson, autovalue and jersey (after an update of dropwizard), and this change makes jackson explicitly fail deserialization rather than silently passing null values to auto-value.